.github/workflows/release-suite-message-system-config.yml

```diff
@@ -0,0 +1,44 @@
+name: "[Release] suite message system config"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
```

Comment:
add empty line after this

Reply:
857865d4ad93a8a7df1123901a12cab8301eb9b6